pyanaconda/modules/storage/partitioning/automatic/automatic_module.py

```diff
@@ -123,6 +125,8 @@ def setup_kickstart(self, data):
         data.autopart.escrowcert = self.request.escrow_certificate
         data.autopart.backuppassphrase = self.request.backup_passphrase_enabled
 
+        data.autopart.hw_passphrase = ""
```

Comment:
May deserve the same comment as the data.autopart.passphrase setting above but feel free to ignore.

Reply:
Ok, I added the comment here as well.